style.css

```diff
@@ -322,85 +323,119 @@ a {
   color: #253858;
 }
 
+.list-lang {
+  display: grid;
+  grid-template-columns: 100px 1fr;
+  list-style-type: none;
+}
+
+.list-lang-item {
+  display: flex;
+  flex-wrap: nowrap;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+  align-items: center;
+}
+
+.list-img {
+  float: left;
+  padding-right: 1rem;
+  padding-bottom: 1rem;
+}
+
 .contact-section {
-  background-image: url('images/contact.png');
-  background-size: cover;
+  background-image: url(images/contac-shape-bg.svg);
   background-repeat: no-repeat;
-  width: 100%;
-  height: auto;
-  flex-direction: column;
-  display: flex;
-  justify-content: center;
-  margin: 0 auto;
+  background-color: #6070ff;
+  background-position: 310px 0;
+  border-top-left-radius: 60px;
```

Comment:
- [ ] **Good job on adding the background image as suggested by the previous reviewer** 👍 👍 

   However, It would be better to adjust the background image to match that of the figma design as much as possible 🙏 

1. 

- [ ] Kindly as suggested by the reviewer, I think it would be better to focus on adjusting your button's design as much as possible to match the original design 👍 



**Your design** | **Figma design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/212940996-570ee257-ee72-4843-a996-68289a1790d9.png) | ![image](https://user-images.githubusercontent.com/102757126/212941363-10a4d7eb-05de-4702-b04e-ccae10b0cef2.png)